src/main/java/com/example/bookshop/dto/BookDto.java

```diff
@@ -0,0 +1,7 @@
+package com.example.bookshop.dto;
+
+import java.math.BigDecimal;
+
+public record BookDto(Long id, String title, String author,
+                      String isbn, BigDecimal price, String description, String coverImage) {

```

Comment:
```suggestion
public record BookDto(Long id, 
                                        String title, 
                                        String author,
                                        String isbn, 
                                        BigDecimal price, 
                                        String description, 
                                        String coverImage
                                        ) {
```
This formatting is more readable